### DISCUSSION TITLE
Not forwarding to payment info after showing booking recorded message

### DISCUSSION THREAD

**Discussion by markrdavidson** [liedekef/events-made-easy#475] [Q&A] [answered]
What things should I check / how can I troubleshoot the problem of the the user not being forwarded to the payment info after being shown the "booking recorded message"?

Thanks!

**Comment by markrdavidson** [comment, marked as answer]
SOLVED: No payment methods had been set for each event. Set payment methods = problem solved.

I WONDER if at least one payment method should be required on the back end? (I'm guessing there are use cases where no payment method is appropriate?)

**Reply by liedekef** [reply]
If only one payment method is configured, that one will be selected automatically for new events. If more than one method is defined, it is up to you to select the method(s) per event.
